tide/wall/TextureNodeYUV.cpp

```diff
@@ -49,11 +49,11 @@ namespace
 
 const char* vertShader =
 R"(
-#version 130
+#version 120
```

Comment:
Yes, that is the problem. The context is created through QOpenGLContext and on Linux you get a GL 4.X context. But to get anything higher than a GL 2 context on OSX you need to create a Core context, and I think this never worked properly. I could give it another try for fun, but it doesn't really matter at this point.